src/Persistence/Csv.php

```diff
@@ -239,7 +239,7 @@ public function typecastLoadRow(Model $model, array $row): array
     public function tryLoad(Model $model, $id): ?array
     {
         if ($id !== self::ID_LOAD_ANY) {
-            throw new \Error/*Exception*/ ('CSV Persistence does not support other than LOAD ANY mode'); // @TODO we can iterate...
+            throw new Exception('CSV Persistence does not support other than LOAD ANY mode'); // @TODO
```

Comment:
lost comment "we can iterate"

Reply:
`we can iterate` was introduced by me yesterday, today I removed it, by `ìterate` I meant another PR which I found a little confusing